server/executor/queue_driver_in_memory.go

```diff
@@ -0,0 +1,61 @@
+package executor
+
+import (
+	"fmt"
+	"log"
+)
+
+type loggerFn func(string, ...any)
+
+func newLoggerFn(name string) loggerFn {
+	return func(format string, params ...any) {
+		log.Printf("[%s] %s", name, fmt.Sprintf(format, params...))
+	}
+}
```

Comment:
This is useful even outside of this package. Perhaps we could move it to a specific infra package in the future?

Reply:
yes, I found the pattern very useful, but if we want to improve our logging I would make it a bit more sophisticated